packages/automerge-repo-storage-nodefs/vitest.config.ts

```diff
@@ -1,8 +1,11 @@
 import { defineConfig, mergeConfig } from "vitest/config"
-import rootConfig from '../../vitest.config'
+import rootConfig from "../../vitest.config"
```

Comment:
Sorry, for missing styling fixes. Looks like I forgot to configure Prettier correctly.